integration_tests/mockApis/community.ts

```diff
@@ -384,17 +384,14 @@ export default {
     })
   },
 
-  stubGetCutOffDateForLicenceTimeOut: (): SuperAgentRequest => {
```

Comment:
Probably obvious but assume we no longer need this or are stubbing hard stop cut off date elsewhere?